src/components/DatePicker/__stories__/datepicker.stories.mdx

```diff
@@ -0,0 +1,95 @@
+import { useState } from "react";
+import moment from "moment";
+import DatePickerComponent from "../DatePicker";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createComponentTemplate, createStoryMetaSettings } from "../../../storybook";
+import {
+  TEXT_FIELD,
+  DROPDOWN,
+  DIALOG_CONTENT_CONTAINER
+} from "../../../storybook/components/related-components/component-description-map";
+import "./DatePicker.stories.scss";
+import DialogContentContainer from "../../DialogContentContainer/DialogContentContainer";
+
+export const metaSettings = createStoryMetaSettings({
+  component: DatePicker
```

Comment:
please pass the names of the function props to the relevant argument (see how we did this in other stories)